cmd/auto-complete.go

```diff
@@ -29,7 +29,31 @@ import (
 // fsComplete knows how to complete file/dir names by the given path
 type fsComplete struct{}
 
-func (fs fsComplete) Predict(a complete.Args) (prediction []string) {
+// predictPathWithTilde completes an FS path which starts with a `~/`
+func (fs fsComplete) predictPathWithTilde(a complete.Args) []string {
+	homeDir := os.Getenv("HOME")
```

Comment:
@vadmeste did you look at my suggestion?

Reply:
> @vadmeste did you look at my suggestion?

done